chatmaild/src/chatmaild/doveauth.py

```diff
@@ -160,6 +160,19 @@ def handle_dovecot_request(msg, db, config: Config):
     return None
 
 
+def handle_dovecot_protocol(rfile, wfile, db: Database, config: Config):
+    while True:
+        msg = rfile.readline().strip().decode()
+        if not msg:
+            break
+        res = handle_dovecot_request(msg, db, config)
+        if res:
+            wfile.write(res.encode("ascii"))
+            wfile.flush()
+        else:
+            logging.warn("request had no answer: %r", msg)
```

Comment:
```suggestion
            logging.warning("request had no answer: %r", msg)
```

currently, pytest shows this warning:

```
src/chatmaild/tests/test_doveauth.py::test_handle_dovecot_protocol
  /home/user/code/chatmail/chatmaild/src/chatmaild/doveauth.py:173: DeprecationWarning: The 'warn' function is deprecated, use 'warning' instead
    logging.warn("request had no answer: %r", msg)
```